### ISSUE TITLE
50% perf hit from 1.3.0 to 2.2.2

### ISSUE THREAD

**Issue by icetbr** [jsmreese/moment-duration-format#107]
Hi, have a simple benchmark test that runs a slow function in my code 20 times. This is the average time:

lib 1.3.0: 169ms
lib 2.2.2: 267ms

Duration.format is called 5397 times.

Just thought I would let you guys know.

**Comment by jsmreese** [comment]
This is true. Version 2 does a lot of extra work in the name of accuracy. JavaScript isn't exactly good at math.

You have a good point. Version 2 seems to have its bugs ironed out now, so it's a good time to look for performance improvements.

> Just thought I would let _you guys_ know.
This made me smile. There's no _you guys_... just me. 😄

**Comment by icetbr** [comment]
Oh, that is so sad to hear :-) Well, thanks for your efforts, keep it up the good work

**Comment by ticky** [comment]
Now that #122 is landed, I’d love to know what the perf numbers are looking like here!
